contracts/protocol/integration/auction-price/BoundedStepwiseExponentialPriceAdapter.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.17;
+
+import { FixedPointMathLib } from "solady/src/utils/FixedPointMathLib.sol";
+
+/**
+ * @title BoundedStepwiseExponentialPriceAdapter
+ * @author Index Coop
+ * @notice Price adapter contract for the AuctionRebalanceModuleV1. It returns a price that
+ * increases or decreases exponentially in steps over time, within a bounded range.
+ * The rate of change is increasing.
+ * Price formula: price = initialPrice +/- scalingFactor * e ^ (timeCoefficient * timeBucket)
+ */
+contract BoundedStepwiseExponentialPriceAdapter {
```

Comment:
The reason I do not have this dependency is because `IAuctionPriceAdapter` is compiled with 0.6.10 so that it can be referenced in `AuctionRebalanceModuleV1`, while the actual adapters are compiled with 0.8.17. 

I agree it would be nice though. Any suggested workaround?